### PR TITLE
Feature/9215 booster notifications dev menu extension

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Add Dev menu extension for booster notification
functionalities:

- select a health certified person
- download the rules
- Show the downloaded booster rules
- Show the passed rule for this person if it existed
- show last download date
- clear the cached last download date
- clear the cached rules
- clear cached passed rule
- Also show errors in alert if any was encountered 

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9215

## Screenshots
https://user-images.githubusercontent.com/15270737/132816079-96db4c8d-6c6d-4fad-ac94-91080cb5dd7c.MP4

### CODE REVIEW

**Comment by 30mar** [comment]
A small note: we download the rules and try to match with all health certified persons, and we show if there is any error, for example in the attached video we had two persons, when we tried to download the rules we get a match for Robert and an error for Erika because Erika is not eligible a for a booster vaccination

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift [hunk 1]
```diff
@@ -171,7 +171,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		updateExposureState(state)
 		Analytics.triggerAnalyticsSubmission()
 		appUpdateChecker.checkAppVersionDialog(for: window?.rootViewController)
-		healthCertificateService.checkIfBoosterRulesShouldBeFetched()
+		healthCertificateService.checkIfBoosterRulesShouldBeFetched(completion: {_ in })
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift:174]: Can we do it more beautiful, like passing a default value or something like that?

File: src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/BoosterNotificationServiceError.swift [hunk 2]
```diff
@@ -8,6 +8,52 @@ import HealthCertificateToolkit
 enum BoosterNotificationServiceError: LocalizedError {
 	case CERTIFICATE_VALIDATION_ERROR(HealthCertificateValidationError)
 	case BOOSTER_VALIDATION_ERROR(BoosterNotificationRuleValidationError)
+
+	// ONLY USED FOR TESTING IN THE DEV MENU
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/BoosterNotificationServiceError.swift:12]: when only used for testing, you could surround it with #if !RELEASE

File: src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift (hunk 1 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift:174]: yeah we can do that I can log the error

**Review by pascal-brause — APPROVED** [review]

**Pull request merged**